receiver/filelogreceiver/README.md

```diff
@@ -1,29 +1,63 @@
+| `multiline`            |                  | A `multiline` configuration block. See below for more details                                                      |
+| `encoding`             | `nop`            | The encoding of the file being read. See the list of supported encodings below for available options               |
+| `include_file_name`    | `true`           | Whether to add the file name as the label `file_name`                                                              |
+| `include_file_path`    | `false`          | Whether to add the file path as the label `file_path`                                                              |
+| `poll_interval`        | 200ms            | The duration between filesystem polls                                                                              |
+| `fingerprint_size`     | `1kb`            | The number of bytes with which to identify a file. The first bytes in the file are used as the fingerprint. Decreasing this value at any point will cause existing fingerprints to forgotten, meaning that all files will be read from the beginning (one time) |
+| `max_log_size`         | `1MiB`           | The maximum size of a log entry to read before failing. Protects against reading large amounts of data into memory |
+| `max_concurrent_files` | 1024             | The maximum number of log files from which logs will be read concurrently. If the number of files matched in the `include` pattern exceeds this number, then files will be processed in batches. One batch will be processed per `poll_interval` |
+| `labels`               | {}               | A map of `key: value` labels to add to the entry's labels                                                          |
+| `resource`             | {}               | A map of `key: value` labels to add to the entry's resource                                                        |
+| `operators`            | []               | An array of [operators](https://github.com/open-telemetry/opentelemetry-log-collection/blob/main/docs/operators/README.md#what-operators-are-available). See below for more details |
 
-## Optional Parameters
+Note that _by default_, no logs will be read from a file that is not actively being written to because `start_at` defaults to `end`.
 
-- `plugin_dir` is the path to a directory which contains `stanza` [plugins](https://github.com/open-telemetry/opentelemetry-log-collection/blob/main/docs/plugins.md#using-plugins). Plugins are parameterized pipelines that are designed for specific use cases.
-- `offsets_file` is the path to a file that `stanza` will use to remember where it left off when reading from files or other persistent input sources. If specified, `stanza` will create and manage this file.
+### Operators
 
-## Operator Basics
+Each operator performs a simple responsibility, such as parsing a timestamp or JSON. Chain together operators to process logs into a desired format.
```

Comment:
Only "parser" and "general purpose" operators make sense to be used, right? Maybe mention that here.